--- m4-1.4.18/lib/c-stack.c	2016-12-31 21:54:41.000000000 +0800
+++ /tmp/c-stack.c.orig	2026-07-27 19:11:21.272524826 +0800
@@ -52,7 +52,7 @@
 #endif
 #ifndef SIGSTKSZ
 # define SIGSTKSZ 16384
-#elif HAVE_LIBSIGSEGV && SIGSTKSZ < 16384
+#elif 0 /* CI: glibc >= 2.34 SIGSTKSZ non-constant */
 /* libsigsegv 2.6 through 2.8 have a bug where some architectures use
    more than the Linux default of an 8k alternate stack when deciding
    if a fault was caused by stack overflow.  */
